src/dbnode/client/fetch_tagged_results_accumulator.go

```diff
@@ -52,9 +59,10 @@ type fetchTaggedResultAccumulator struct {
 	numHostsPending         int32
 	numShardsPending        int32
 
-	errors     xerrors.Errors
-	responses  fetchTaggedIDResults
-	exhaustive bool
+	errors       xerrors.Errors
+	responses    fetchTaggedIDResults
```

Comment:
super nit: mind renaming var to `fetchResponses` or something

Reply:
Sure thing.